backend/fms_core/viewsets/_constants.py

```diff
@@ -176,7 +176,7 @@
     "status": CATEGORICAL_FILTERS,
     "external_id": CATEGORICAL_FILTERS,
     "external_name": CATEGORICAL_FILTERS,
-    **_prefix_keys("project_derived_samples__samples__", _sample_minimal_filterset_fields),
+    **_prefix_keys("project_derived_by_samples__sample__", _sample_minimal_filterset_fields),
```

Comment:
Why does this in particular have only a single underscore? That feels off...

Reply:
This is the related_name for that model attribute ...